generate_readme/data.yaml

```diff
@@ -5,6 +5,7 @@
 # Contributors: If you want to propose a new category, add it to the array below.
 items_headline_genres:
 - "Communication - Social Networks and Forums"
+- "Money, Budgeting & Management"
```

Comment:
refer to https://github.com/awesome-selfhosted/awesome-selfhosted?tab=readme-ov-file#money-budgeting--management